### DISCUSSION TITLE
Change passphrase

### DISCUSSION THREAD

**Discussion by nic** [krzyzanowskim/ObjectivePGP#216] [General]
Is there any way to change the passphrase of an existing private key?

Like the style we got on the shell:
```
gpg --edit-key Your-Key-ID-Here
gpg> passwd
gpg> save
```

Keeping the key's fingerprint, only changing the passphrase.

**Comment by krzyzanowskim** [comment]
Not yet. The key edit feature is coming in next weeks.

**Comment by nic** [comment]
@krzyzanowskim Awesome! Thanks :)

**Comment by kristijanPetr** [comment]
Hi  , 
Is it now change passphrase feature implemented to the Library ?

**Comment by danielfttorres** [comment]
Hey @krzyzanowskim. How I can change the _key passphrase_? This method is available?

Thanks!

**Comment by krzyzanowskim** [comment]
This feature is not available

**Comment by juangarciadev** [comment]
Hey @krzyzanowskim. Do you know how I can change the private key passphrase? Thanks
